content/docs/1.9.0/concepts.md

```diff
@@ -98,7 +98,7 @@ The Kubernetes CSI plugin calls Longhorn to create volumes to create persistent
 
 The Kubernetes cluster internally uses the CSI interface to communicate with the Longhorn CSI plugin. And the Longhorn CSI plugin communicates with the Longhorn Manager using the Longhorn API.
 
-Longhorn does leverage iSCSI, so extra configuration of the node may be required. This may include the installation of open-iscsi or iscsiadm depending on the distribution.
+For V1 volumes, Longhorn relies on iSCSI, which may require additional node setup, including the installation of `open-iscsi` or `iscsiadm`, depending on the distribution. V2 volumes, on the other hand, have different prerequisites, such as the need for specific kernel modules and huge pages.
```

Comment:
Should we mention NVMe or ublk frontend?

Reply:
Updated. Thank you.